src/parse/parse_mat.jl

```diff
@@ -0,0 +1,1667 @@
+# CONTRIB file
+# Originally from PowerModels.jl. Adapted by PowerSystems.jl and by us.
```

Comment:
I would maybe add a link to the original parser in PowerModels.jl

Reply:
Yes. Done.